apps/dashboard/src/utils/clerk-appearance.ts

```diff
@@ -0,0 +1,23 @@
+import { SignUpTheme, SignInTheme } from '@clerk/types';
+
+export const clerkAppearance: SignUpTheme | SignInTheme = {
```

Comment:
A more accurate name would be:

```suggestion
export const clerkSignupAppearance: SignUpTheme | SignInTheme = {
```